components/script/dom/eventdispatcher.rs

```diff
@@ -105,14 +109,17 @@ fn dispatch_to_listeners(event: &Event, target: &EventTarget, chain: &[&EventTar
     assert!(!event.stop_propagation());
     assert!(!event.stop_immediate());
 
-    /* bubbling */
+    // Step 8.
     if !event.bubbles() {
         return;
     }
 
```

Comment:
The comment is missing, but I believe it's still being done when we build the chain it's in tree order ( https://dom.spec.whatwg.org/#concept-tree-order ) here https://github.com/servo/servo/pull/9409/files#diff-e70183cc533bf121ea8a9c077ea36d90R159 and when we do the capturing phase it's in reverse tree order https://github.com/servo/servo/pull/9409/files#diff-e70183cc533bf121ea8a9c077ea36d90L68. I am not sure the best way to document this.